core/src/components/button/button.ios.vars.scss

```diff
@@ -108,7 +108,7 @@ $button-ios-small-height:                             2.1em !default;
 $button-ios-small-border-radius:                      6px !default;
 
 /// @prop - Font size of the small button
-$button-ios-small-font-size:                          13px !default;
+$button-ios-small-font-size:                          min(0.8125rem, 39px) !default;
```

Comment:
I used 3x the value of the `font-size` at `100%` to calculate the `max`. In this case `13px * 3`.